### PR TITLE
Accessibility: Fix  text contrast errors on certain elements

### PR DESCRIPTION
**Pull request by LaurenQurashi**
## Description of change

As part of the Accessibility work, a small sample of pages have been tested for Accessibility issues before we get them audited. This PR focuses on changing the colours and styles of some elements which flagged up as not having a significant enough text contrast. 

The elements that have been amended are: 

- The hint text on the collection filters that sit on the right hand side of the Companies (and other) pages. It has been darkened to stand out more against it's grey background.

- The Active filters that sit in the collection header in the Companies (and other) pages. It has been made to look a touch more like the GOVUK filter tags, with a thin black border, a light grey background, and had the tag label text darkened so that it stands out more. 

## Test instructions

You should be able to navigate to any of the tabs within the grey navigation banner, and see that the hint text in the filters is #505A5F. You should also be able to navigate to the companies page and apply a filter. There you will see it's new design of a background of #F3F2F1, and a 1px wide solid border. 

The visual tests have been updated for this PR, however please pay attention to these as It's the first time I've run / updated visual tests! 

## Screenshots

Filter hint text before: 

Screenshot 2020-09-15 at 16 57 48

Filter hint text after: 

Screenshot 2020-09-15 at 16 46 28

Filter tags before: 

Screenshot 2020-09-15 at 16 57 55

Filter tags after: 

Screenshot 2020-09-21 at 11 51 32

Interaction design requested that we try to make the filter tags look closer to the GOV.UK ones like so: 

Screenshot 2020-09-21 at 11 26 21

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/templates/_macros/collection-header/_collection-header.scss [hunk 1]
```diff
@@ -51,18 +51,21 @@
 }
 
 .c-collection__filter-tag {
-  color: $text-colour;
   display: inline-block;
   position: relative;
+  border-style: solid;
+  border-color: black;
```
Comment (paulgain) [inline src/templates/_macros/collection-header/_collection-header.scss:57]: Can we go with `#505A5F` here? Black seems too bold a colour IMO. Using `#505A5F` is more subtle, I think the contrast would still there, although you would need to check.

**Comment by paulgain** [comment]
I think before and after screenshots would be helpful in the future, but looking good. 👍

File: src/templates/_macros/collection-header/_collection-header.scss (hunk 1 above)
Comment (mforner13) [reply src/templates/_macros/collection-header/_collection-header.scss:57]: I agree with Paul here. Will be good to check with Pauline to make sure she'll still happy with that contrast :)

Comment (LaurenQurashi) [reply src/templates/_macros/collection-header/_collection-header.scss:57]: I'll ask her tomorrow and see what she says, as I think they are trying to move us towards looking a bit more like GOV.UK? but I shall ask 😎  thanks!

Comment (ian-leggett) [reply src/templates/_macros/collection-header/_collection-header.scss:57]: I would use BLACK from - https://github.com/penx/govuk-colours/blob/master/src/index.js

Comment (LaurenQurashi) [reply src/templates/_macros/collection-header/_collection-header.scss:57]: Pauline has asked for the border to be changed to #505A5F so I'll make those changes now 🎉

**Review by mforner13 — APPROVED** [review]

**Review by paulgain — APPROVED** [review]
LGTM 👍

**Comment by paulinewongDIT** [comment]
@LaurenQurashi bringing/copying the slack comment here so all the feedback is documented in one place: 
"Thanks  for the update. Yep please update the border colour as per GOV UK search filter “facet-tag” border colour. In screenshot attached, I have checked the gov border colour  - it is #505A5F. ([url](url
Screenshot 2020-09-22 at 09 13 09

**Pull request merged**